test/e2e/storage/persistent_volumes.go

```diff
@@ -220,8 +220,8 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 		Context("with multiple PVs and PVCs all in same ns", func() {
 
 			// define the maximum number of PVs and PVCs supported by these tests
-			const maxNumPVs = 10
-			const maxNumPVCs = 10
+			//const maxNumPVs = 10
```

Comment:
Well, if the constants are really unused then please remove them instead of commenting out.